tests/sqlparser_common.rs

```diff
@@ -7118,3 +7122,30 @@ fn parse_create_type() {
         create_type
     );
 }
+
+#[test]
+fn parse_range_select() {
```

Comment:
Add some cases:
- the same aggregator with `RANGE` and without `RANGE` in one query
- `FILL` without `RANGE`
-  `FILL ... FILL ...` and `ALIGN ... ALIGN ...`